frontend/packages/component-library/src/common/contexts/ThemeContext.tsx

```diff
@@ -63,7 +63,9 @@ export const ThemeProvider = ({children}: {children: ReactNode}) => {
    * */
   return (
     <ThemeContext.Provider value={{theme, toggleTheme, setTheme}}>
-      <div data-theme={theme}>{children}</div>
+      <div data-theme={theme} style={{width: '100%'}}>
```

Comment:
Added this because an issue was happening in Contentful where it was only applying styles to the section's container and not the entire width of the screen.